accelerator/cli_info.h

```diff
@@ -90,6 +91,7 @@ static struct ta_cli_argument_s {
     {"no-gtta", no_argument, NULL, NO_GTTA, "Disable getTransactionToConfirm (gTTA) when sending transacation"},
     {"buffer_list", required_argument, NULL, BUFFER_LIST, "Set the value of `buffer_list_name`"},
     {"done_list", required_argument, NULL, DONE_LIST, "Set the value of `done_list_name`"},
+    {"cache_capacity", required_argument, NULL, CACHE_CAPACITY, "Set the maximum redis capacity`"},
```

Comment:
Don't mention Redis here. We would like to encapsulate between CLI operations and implementation details.